### ISSUE TITLE
HTML over FTP not always reachable

### ISSUE THREAD

**Issue by erkin** [mozilla-mobile/focus-android#4345]
## Steps to reproduce
Navigate to an HTML page over FTP.

### Expected behavior
The page renders successfully.

### Actual behavior
Depends.

#### On WebView-based Focus/Klar
  * All FTP hyperlinks are ignored (ie pressing them does nothing).

#### On GeckoView-based Focus/Klar
  * Access through a hyperlink in a web page: The page renders correctly, without any problem.
  * Access through a hyperlink in an FTP page: The action is ignored. (However, HTTP hyperlinks in FTP pages work.)
  * Access by opening a hyperlink in a web page in a new tab: The page renders correctly.
  * Access by opening a hyperlink in an FTP page in a new tab: Firefox loads `about:blank` on the new tab.
  * Access by successfully navigating to an FTP page, then away to a web page, then pressing back: Firefox ignores the action and remains in the web page after the FTP page in history.
  * Direct access (ie entering the FTP URL in the address bar, or through a shortcut on Android launcher): Firefox loads `about:blank`.

In short, an FTP page is only accessible through a web page.

### Device information
* Android device: Meizu Note 8, Android 8.1.0
* Focus version: 8.0.16

**Comment by lobontiumira** [comment]
I was not able to reproduce this issue on the latest Focus Nightly 99.0a1 build from 3/8 with Lenovo M10 tablet (Android 10).

@erkin could you please retest?
Thank you!

**Comment by ionutbedregeanu** [comment]
I will close this ticket since can't be reproduced anymore. @erkin Feel free to reopen it if it's still reproducible
